sync/notify/msg.py

```diff
@@ -109,11 +109,14 @@ def summary_message(results):
         data[-1] += " and %s subtests" % subtests
     data[-1] += "\n"
 
+    result_statuses = [status for status in statuses if status in summary]
+    if not result_statuses:
+        return data
+
     data.append("## Status Summary\n")
 
-    max_width = max(len(status) for status in statuses if status in summary)
+    max_width = max(len(status) for status in result_statuses)
```

Comment:
There is another way to do that:

```suggestion
    max_width = len(max(result_statuses, key=len))
```

Note that it matters very much given the nano seconds nature. :)

```
% python -m timeit 'len(max(["a", "bcd", "ef", "ghij", "kl"], key=len))'            
500000 loops, best of 5: 654 nsec per loop
% python -m timeit 'max(len(status) for status in ["a", "bcd", "ef", "ghij", "kl"])'
500000 loops, best of 5: 878 nsec per loop
```

Reply:
Oh neat, I didn't know that worked.